### ISSUE TITLE
Origami prepare command throwing error with Polymer v3.3.1

### ISSUE THREAD

**Issue by nagarajhubli** [hotforfeature/origami#98]
Polymer released [v3.3.1](https://github.com/Polymer/polymer/releases/tag/v3.3.1) around a week ago, and Origami prepare command is throwing error with the latest version of Polymer. Please find the error message below.

```
Origami: Error: ENOENT: no such file or directory, scandir '/Users/nagarajh/repos/bitbucket/angular7-polymer3/node_modules/@polymer/polymer/_origami-es5'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angular7-polymer3@0.0.0 prepare:es5: `origami prepare es5 node_modules/{nd-ui-components/,paper-collapse-item/,paper-range-slider/,@polymer/*,@vaadin/*,@webcomponents/shadycss}`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the angular7-polymer3@0.0.0 prepare:es5 script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nagarajh/.npm/_logs/2019-11-15T06_19_23_238Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angular7-polymer3@0.0.0 start: `npm run prepare:es5 && ng serve angular7-polymer3 --host 0.0.0.0`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the angular7-polymer3@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nagarajh/.npm/_logs/2019-11-15T06_19_23_257Z-debug.log

```

**Comment by asyncLiz** [comment]
Thanks for the bug! This should be fixed in Origami 3.1.2.

**Comment by nagarajhubli** [comment]
Thanks for the quick fix.
